src/hotspot/share/gc/g1/g1Policy.cpp

```diff
@@ -628,11 +629,12 @@ double G1Policy::logged_cards_processing_time() const {
 // Anything below that is considered to be zero
 #define MIN_TIMER_GRANULARITY 0.0000001
 
-void G1Policy::record_collection_pause_end(double pause_time_ms, bool concurrent_operation_is_full_mark) {
+void G1Policy::record_young_collection_end(bool concurrent_operation_is_full_mark) {
   G1GCPhaseTimes* p = phase_times();
 
-  double end_time_sec = os::elapsedTime();
-  double start_time_sec = phase_times()->cur_collection_start_sec();
+  double start_time_s = phase_times()->cur_collection_start_sec();
```

Comment:
I dislike this change of nomenclature to use `_s` suffix instead of `_sec` suffix to indicate units of seconds.  The latter is in wide use in GC code.